README.md

```diff
@@ -37,31 +37,60 @@ Here's the steps to set up the project locally:
 2. `poetry install`
 3. `make up`
 4. `make migrate`
-5. `make load_fixtures`
-6. `make createuser`
+5. `make loadfixtures`
+6. `make creategroups`
```

Comment:
group creation and handling must not be included here, for the time being we can perform those steps manually.